focus-android/quality/detekt.yml

```diff
@@ -1,51 +1,70 @@
+# Please refer to https://github.com/mozilla-mobile/firefox-android/blob/main/android-components/config/detekt.yml
+# for the source of truth for our detekt configuration.
+
 build:
   maxIssues: 0
+  excludeCorrectable: false
   weights:
-    # complexity: 2
-    # LongParameterList: 1
-    # style: 1
-    # comments: 1
+  # complexity: 2
+  # LongParameterList: 1
+  # style: 1
+  # comments: 1
 
 processors:
   active: true
   exclude:
+  # - 'DetektProgressListener'
+  # - 'KtFileCountProcessor'
+  # - 'PackageCountProcessor'
+  # - 'ClassCountProcessor'
   # - 'FunctionCountProcessor'
   # - 'PropertyCountProcessor'
-  # - 'ClassCountProcessor'
-  # - 'PackageCountProcessor'
-  # - 'KtFileCountProcessor'
+  # - 'ProjectComplexityProcessor'
+  # - 'ProjectCognitiveComplexityProcessor'
+  # - 'ProjectLLOCProcessor'
+  # - 'ProjectCLOCProcessor'
+  # - 'ProjectLOCProcessor'
+  # - 'ProjectSLOCProcessor'
+  # - 'LicenseHeaderLoaderExtension'
 
 console-reports:
   active: true
   exclude:
-  #  - 'ProjectStatisticsReport'
-  #  - 'ComplexityReport'
-  #  - 'NotificationReport'
-  #  - 'FindingsReport'
-  #  - 'BuildFailureReport'
-  #  - 'HtmlOutputReport'
-    - 'PlainOutputReport'
-    - 'XmlOutputReport'
+    #  - 'ProjectStatisticsReport'
+    #  - 'ComplexityReport'
+    #  - 'NotificationReport'
+    #  - 'FindingsReport'
+    #  - 'FileBasedFindingsReport'
+    - 'LiteFindingsReport'
 
 comments:
   active: true
-#  excludes: "**/*Test.kt, **/*Spec.kt, **/test/**, **/androidTest/**"
   AbsentOrWrongFileLicense:
-    active: true
```

Comment:
This will be a notable change because in 1.19.0 this rule changed to expects a template file to match the license. https://detekt.dev/docs/rules/comments#absentorwrongfilelicense

We can address re-enabling this rule and others as follow up after https://github.com/mozilla-mobile/firefox-android/pull/292